examples/example.py

```diff
@@ -60,7 +59,7 @@ def __init__(self, size=(600, 600), hex_radius=22, caption="ExampleHexMap"):
         self.hex_apothem = hex_radius * np.sqrt(3) / 2
         self.hex_offset = np.array([self.hex_radius * np.sqrt(3) / 2, self.hex_radius])
 
-        self.hex_map = hx.HexMap()
+        self.hex_map = HexMap()
```

Comment:
You've got `import hexy as hx` on line 6, so `hx.HexMap()` shouldn't be broken. I don't see the need for this particular change.